src/handlers/track_table.rs

```diff
@@ -252,6 +253,16 @@ fn play_random_song(app: &mut App) {
   };
 }
 
+fn handle_save_track_event(app: &mut App) {
+  let (selected_index, tracks) = (&app.track_table.selected_index, &app.track_table.tracks);
+  if let Some(track) = tracks.get(*selected_index) {
+    if let Some(id) = &track.id {
+      let _id = id.to_string();
+      app.dispatch(IoEvent::ToggleSaveTrack(_id));
```

Comment:
The underscore prefix (`_id`) is usually used to convey that the variable is not used.

Instead you could do this?

```suggestion
      app.dispatch(IoEvent::ToggleSaveTrack(id.to_string()));
```

Reply:
Good to know the underscore means unused (Bad habit from other languages, where the underscore prefix is for a private/local variable).

Your suggestion is what I had in the original commit, which broke the build checks.

Reply:
Ah okay cool, that's all good then!

So if you could just remove the underscore then we are good to go!

Reply:
Fixed up and also merged in latest master changes. Works locally. So hoping all Github checks still pass.